shell/platform/common/cpp/text_input_model_unittests.cc

```diff
@@ -103,8 +103,8 @@ TEST(TextInputModel, AddCodePointSelectionWideCharacter) {
 TEST(TextInputModel, AddText) {
   auto model = std::make_unique<TextInputModel>("", "");
   model->AddText(u"ABCDE");
-  model->AddText(u"😄");
-  model->AddText(u"FGHIJ");
```

Comment:
Why are you (inconsistently) removing the unicode marker?

Reply:
This is because the tests were originally written with both AddText() methods in mind. I then split the tests out into https://github.com/flutter/engine/pull/18411, so this PR is returning the test to the original case of mixing both methods.

Reply:
Right, I forgot `u` is UTF-16, not UTF-8.

I would much rather have `AddTextUtf16` and `AddTextUtf8` methods that are identical except for which `AddText` they call, rather than removing the coverage of surrogate pair handling in the UTF-16 path as this change is doing.